test/index.js

```diff
@@ -567,6 +567,31 @@ test ('Contravariant', () => {
 });
 
 test ('equals', () => {
+  const {nestedSetoidArb} = jsc.letrec (tie => ({
+    builtinSetoidArb: jsc.record ({
+      a: tie ('nestedSetoidArb'),
+    }),
+    noSetoidArb: jsc.record ({
+      'a': tie ('nestedSetoidArb'),
+      '@@type': jsc.constant ('sanctuary-type-classes/NoSetoid@1'),
+      '@@show': jsc.constant (function() { return `NoSetoid (${show (this.a)})`; }),
+    }),
+    customSetoidArb: jsc.record ({
+      'a': tie ('nestedSetoidArb'),
+      '@@type': jsc.constant ('sanctuary-type-classes/CustomSetoid@1'),
+      '@@show': jsc.constant (function() { return `CustomSetoid (${show (this.a)})`; }),
+      'fantasy-land/equals': jsc.constant (function(that) { return Z.equals (this.a, that.a); }),
+    }),
+    nestedSetoidArb: jsc.oneof ([
+      tie ('builtinSetoidArb'),
+      tie ('noSetoidArb'),
+      tie ('customSetoidArb'),
+      jsc.nat,
+    ]),
+  }));
+
+  nestedSetoidArb.show = show;
```

Comment:
What does this line do?

Reply:
It overrides the built-in `show` method provided by jsverify, which is just JSON.stringify, so that if an error occurs, the output shows the two inequal values nicely.